x-pack/legacy/plugins/siem/public/components/timeline/body/events/stateful_event.tsx

```diff
@@ -98,69 +110,91 @@ export class StatefulEvent extends React.Component<Props, State> {
       timelineId,
       toggleColumn,
       updateNote,
-      width,
     } = this.props;
 
     // If we are not ready to render yet, just return null
     // see componentDidMount() for when it schedules the first
     // time this stateful component should be rendered.
     if (!this.state.initialRender) {
-      return null;
+      return <EmptyRow style={{ height: '27px' }}></EmptyRow>;
```

Comment:
nit: consider replacing `27px` with a constant that includes a `jsdoc` explaining that it should be adjusted if we change the empty row hight